ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-hosted-pipeline.yml

```diff
@@ -441,28 +442,29 @@ spec:
           workspace: repository
           subPath: src
 
-    # TODO: passing kubeconfig here is just a temporary workaround to make the task pass
     - name: preflight
       runAfter:
         - ocp-environment-preparation
       taskRef:
-        name: preflight
+        kind: Task
+        name: preflight-trigger
       params:
```

Comment:
AFAIK the preflight should run for the latest supported version of the operator:

```
        - name: ocp_version
          value: "$(tasks.get-supported-versions.results.max_supported_ocp_version)"
```

Reply:
See https://github.com/redhat-openshift-ecosystem/operator-pipelines/pull/67#discussion_r719554484